frontend/taipy-gui/src/components/Taipy/TableFilter.spec.tsx

```diff
@@ -48,7 +48,8 @@ beforeEach(() => {
 });
 
 afterEach(() => {
-    // @ts-ignore
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
❓

Reply:
I was trying to fix linter issue, but I guess this was not the problem.

Reply:
linter can be usefull ...